dotnet/src/SemanticKernel/Planning/Planners/PlannerConfig.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All not reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Planning.Planners;
+
+/// <summary>
+/// Common configuration for planner instances.
+/// </summary>
+public sealed class PlannerConfig
```

Comment:
I was on board for getting rid of planner skill, but now I see it more as Removing FunctionFlowRunner and replacing with FunctionFlowParser to take NL XML and convert to a Plan object and then rely on native kernel support for execution. The PlannerSkill can still expose this and use the FunctionFlowPlanner just like a developer could use directly, as well.